src/pages/de/core-concepts/astro-components.md

```diff
@@ -0,0 +1,400 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Komponenten
+description: Eine Einführung in die Syntax der .astro-Komponenten.
+---
+
+**Astro-Komponenten** sind die Grundbausteine eines jeden Astro-Projekts. Sie sind reine HTML-Templatekomponenten ohne clientseitige Laufzeit.
```

Comment:
```suggestion
**Astro-Komponenten** sind die Grundbausteine eines jeden Astro-Projekts. Sie sind reine HTML-Vorlagen ohne clientseitigen Laufzeit-Code.
```

Oops, missed that one "Template" at first! Shortened to "Vorlagen" instead of "Vorlage(n?)komponenten" because it's clear from the context that we're talking about components.

I'm also suggesting to add "-Code" to "Laufzeit" because the German translation of "runtime" isn't as clear otherwise IMHO.